templates/_macros/cta/cta-block.jinja

```diff
@@ -0,0 +1,22 @@
+# Params
+# primary_text: primary CTA button text, shown in a primary-colored button
+# primary_url: primary CTA button URL
+# secondary_text: secondary CTA button text, shown as a link
+# secondary_url: secondary CTA button URL
+{% macro cta_block(
+  primary_text = "Learn more",
```

Comment:
I don't believe so, but now I'm recalling the preference to use caller "slots" for this sort of thing and use params for flags. I'll try that out for consistency's sake - though I was running into issues calling the CTA block macro from within the hero macro earlier:
```
macro 'cta_block' was invoked with two values for the special caller argument. This is most likely a bug.
```
I'll give it another shot

Reply:
Fixed - I just hadn't properly implemented `caller`, as you said - thanks :)
I also switched the CTA block macro to use caller for passing in the button/link content, rather than caller. I kept the href as a parameter since that seems to make more sense.